debian/odoo.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Odoo Open Source ERP and CRM
+After=network.target
+
+[Service]
+Type=simple
+User=odoo
+Group=odoo
+ExecStart=/usr/bin/odoo --config /etc/odoo/odoo.conf --logfile /var/log/odoo/odoo-server.log
+
```

Comment:
I suggest adding [`KillMode=mixed`](https://www.freedesktop.org/software/systemd/man/systemd.kill.html#KillMode=).

Reply:
Ok, it makes sense.